doc/api/net.md

```diff
@@ -40,8 +40,9 @@ added: v0.1.90
 
 * {Error}
 
-Emitted when an error occurs.  The [`'close'`][] event will be called directly
-following this event.  See example in discussion of `server.listen`.
+Emitted when an error occurs. Unlike [`net.Socket`][], the [`'close'`][]
+event will NOT be called directly following this event, unless you call
+`server.close()` manually.  See example in discussion of `server.listen`.
```

Comment:
Thanks. 

The net.Socket is not an event, so can not be displayed as a link by [`'net.Socket'`][], this is why i just use [`net.Socket`][].

Besides, "See the example in the discussion", two "the" seems odd? So i stripped second "the".